src/main/java/it/gov/pagopa/payhub/activities/utility/Utilities.java

```diff
@@ -0,0 +1,45 @@
+package it.gov.pagopa.payhub.activities.utility;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Utilities {
+
+    private Utilities(){}
+    public static final String EMAIL_PATTERN = "^[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$"; //RFC 5322
+    public static final int IBAN_LENGTH = 27;
+
+    public static boolean isValidEmail(final String email) {
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);
```

Comment:
the compile operation could be a static operation, let's introduce a constant to perform this just once